gallery/src/styles/NxCard/NxCardVerticalLayoutExample.tsx

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine, faEdit, faTrash, faCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardRowLayoutExample() {
+  return (
+    <>
+      <div className="nx-card-container nx-card-container--row">
```

Comment:
This example has a mix of separate `.nx-card-container--row`s and `.nx-card-container--row`s that wrap.  I think we should clarify that difference in some way.  Either by splitting it into multiple examples or by putting some label text above each `nx-card-container--row` describing it.

Reply:
> This example has a mix of separate `.nx-card-container--row`s and `.nx-card-container--row`s that wrap. I think we should clarify that difference in some way. Either by splitting it into multiple examples or by putting some label text above each `nx-card-container--row` describing it.

This is something that I was going to bring up in WW today, but unfortunately @dturner32 won't be there. If we're moving to a broader responsive model then things like this are going to wrap more frequently at places we can't always anticipate.